server/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -58,24 +62,28 @@ public final class ProgramIndexView extends BaseHtmlView {
   private final AdminLayout layout;
   private final String baseUrl;
   private final ProgramCardFactory programCardFactory;
+  private final ProgramService programService;
   private final SettingsManifest settingsManifest;
 
   @Inject
   public ProgramIndexView(
       AdminLayoutFactory layoutFactory,
       Config config,
       SettingsManifest settingsManifest,
-      ProgramCardFactory programCardFactory) {
+      ProgramCardFactory programCardFactory,
+      ProgramService programService) {
     this.layout = checkNotNull(layoutFactory).getLayout(NavPage.PROGRAMS);
     this.baseUrl = checkNotNull(config).getString("base_url");
     this.programCardFactory = checkNotNull(programCardFactory);
+    this.programService = checkNotNull(programService);
     this.settingsManifest = checkNotNull(settingsManifest);
   }
 
   public Content render(
       ActiveAndDraftPrograms programs,
       ReadOnlyQuestionService readOnlyQuestionService,
       Http.Request request,
+      String selectedStatus,
```

Comment:
**suggestion** If we change this to type `ProgramTab` then the name can also just be something like `activeTab`

Reply:
hmm how's `selectedTab`? I think `activeTab` might be confusing cuz it could be interpreted as the tab that contains active programs or smth.

Reply:
oh good point, sgtm